src/target_intacct/employee_rate.py

```diff
@@ -40,8 +40,10 @@ def employee_rate_upload(intacct_client) -> None:
         )
 
     for index, row in data_frame.iterrows():
-        if row["employeeid"] in ids:
-            month, day, year = row["ratestartdate"].split("/")
+        if str(row["employeeid"]) in ids:
+            date = parse(row["ratestartdate"])
+            formatted_date = date.strftime('%m/%d/%Y')
+            month, day, year = formatted_date.split("/")
```

Comment:
Can we have this be `year, month, day = start_date.year, start_date.month, start_date.day` and not use `split()` at all?